Documentation/proposals/202411-zone-aware-sharding.md

```diff
@@ -0,0 +1,357 @@
+# Zone aware sharding for prometheus
+
+* **Owners:**
+  * [arnecls](https://github.com/arnecls)
+
+* **Related Tickets:**
+  * [#6437](https://github.com/prometheus-operator/prometheus-operator/issues/6437)
+
+* **Other docs:**
+  * [Well known kubernetes labels](https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone)
+  * [AWS zone names](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones)
+  * [GCP zone names](https://cloud.google.com/compute/docs/regions-zones#available)
```

Comment:
It would be good to mention the shard autoscaling proposal. I don't think that it has a significant impact on this proposal but it should mentioned as a generic item in the goals (e.g. `Compatibility with the Shard autoscaling functionality`).
```suggestion
  * [GCP zone names](https://cloud.google.com/compute/docs/regions-zones#available)
  * [Shard Autoscaling](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/proposals/202310-shard-autoscaling.md) design proposal.
```

Reply:
cc @nicolastakashi